### ISSUE TITLE
Heading elements are not in a sequentially-descending order

### ISSUE THREAD

**Issue by albertothedev** [albertothedev/portfolio#21]
In the projects section, the title "Some of my projects" is using an `h1` tag but the next and only other headers, the card titles, are using `h3` tags. They should be changed to use `h2`.
